docker/build/run

```diff
@@ -29,12 +29,29 @@ except subprocess.CalledProcessError:
     del_group = False
 
 try:
-    subprocess.check_call(['useradd', '--groups=docker,sudo', '--shell=/bin/bash', '--uid=' + os.environ['USER_ID'], '--gid=' + os.environ['GROUP_ID'], os.environ['USER_NAME']])
+    subprocess.check_call([
+        'useradd',
+        '--groups=docker,sudo',
+        '--shell=/bin/bash',
+        '--uid=' + os.environ['USER_ID'],
+        '--gid=' + os.environ['GROUP_ID'],
+        os.environ['USER_NAME']
+    ])
 except subprocess.CalledProcessError:
     del_user = False
 
+# Needed wen we run in docker-compose
+if not os.path.exists('/home/' + os.environ['USER_NAME']):
+    os.mkdir('/home/' + os.environ['USER_NAME'])
+    os.chown('/home/' + os.environ['USER_NAME'], int(os.environ['USER_ID']), int(os.environ['GROUP_ID']))
```

Comment:
be careful !!! you mount the host users's home dir in docker-run.py

Reply:
Yes, I have an if not folder exists...